### PR TITLE
font size control #4094

### PR DESCRIPTION
**Pull request by chrlschwb**
Font text too small? #4094

### CODE REVIEW

File: packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx [hunk 1]
```diff
@@ -11,11 +11,11 @@ export interface MarkdownPreviewStylesProps {
...
   switch (size) {
     case 'xs':
-      return '12px'
-    case 's':
       return '14px'
+    case 's':
+      return '18px'
     default:
-      return '16px'
+      return '20px'
```
Comment (traumschule) [inline packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx:18]: Maybe a bit much? Also text in tables is different now.

fontsize

Probably hard to find one setting that fits all. Other pages use a dropdown to change the font size depending on user needs. Or something PDF viewers have:
size

Comment (chrlschwb) [reply packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx:18]: thanks, I will fix the table font issue in this PR.
https://github.com/Joystream/pioneer/pull/4265

Let's merge this one and create a new issue for font size selection dropdown.

Comment (traumschule) [reply packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx:18]: Ok. I assume it's still too big for some users. Also we are wasting space left and right, especially a problem on mobile (=> followup).

font

**Review by traumschule — APPROVED** [review]
Much better already.

**Comment by thesan** [comment]
@chrlschwb I added a https://github.com/Joystream/pioneer/issues/4094#issuecomment-1490021237 on the issue. Feel free to respond there.

**Pull request closed without merging**